boltstub/scripting.py

```diff
@@ -81,24 +82,35 @@ def auto_match(self, tag):
     def on_auto_match(self, request):
         raise NotImplementedError
 
+    def _get_version(self, request, n):
+        minorindex = 4 + (n * 4) + 2
```

Comment:
I could be a good idea put this magic numbers in some constants to make it easier to understand what's happening.